.github/dependabot.yml

```diff
@@ -15,7 +15,7 @@ updates:
   
   # Maintain dependencies for Python packages
   - package-ecosystem: "pip"
-    directory: "/oidc-controller"
+    directory: "/"
```

Comment:
Good catch, that must be it. 
Do you think there's an ignore that would be worth adding here (if that's even possible for this dependabot yml syntax), as the checks below it go into `demo` and `docker` so might be worth having this one avoid those folders that are already scanned with a different ecosystem? I know it's specifying pip so probably won't find any python in there anyways...
Could be overkill unless it's an easy pattern